README.md

```diff
@@ -108,13 +77,7 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [Live Demo Link](replace-with-your-deployment-URL)
 
```

Comment:
- [ ] Kindly remove this link as suggested by the previous reviewer that if you don't have any link right now then kindly delete Live Demo section which you did 👏 but i guess you missed the broken live link 😨 Kindly delete it too to make your readme look clean and professional 😊